ansible/playbook.yml

```diff
@@ -1,5 +1,5 @@
 - hosts: geoservices
```

Comment:
unfortunately this is not how ansible works - we can use the inventory to put whatever hosts we want into the geoservices group,  but the plays must have statically defined host groups.